Sources/Benchmark/Benchmark.swift

```diff
@@ -14,23 +14,31 @@
 
 public protocol AnyBenchmark {
     var name: String { get }
+    var settings: [BenchmarkSetting] { get }
     func run()
 }
 
 internal class ClosureBenchmark: AnyBenchmark {
     let name: String
+    let settings: [BenchmarkSetting]
     let closure: () -> Void
 
-    init(_ name: String, _ closure: @escaping () -> Void) {
+    init(_ name: String, settings: [BenchmarkSetting], using f: @escaping () -> Void) {
```

Comment:
```suggestion
    init(_ name: String, settings: [BenchmarkSetting], closure: @escaping () -> Void) {
```

I don't thing `using` helps clarify @ the call site. WDYT?

Reply:
Sounds good to me!